src/main/java/com/softserve/academy/spaced/repetition/controller/CourseController.java

```diff
@@ -56,105 +55,115 @@
-    public void updateCourse(@PathVariable Long course_id, @Validated(Request.class) @RequestBody Course course) {
-        courseService.updateCourse(course_id, course);
+    @PutMapping(value = "/api/cabinet/courses/{courseId}")
+    public ResponseEntity<CoursePublicDTO> updateCourse(@PathVariable Long courseId, @Validated(Request.class) @RequestBody Course course) {
+        courseService.updateCourse(courseId, course);
+        Link selfLink = linkTo(methodOn(CourseController.class).getCourseById(course.getId())).withSelfRel();
+        CoursePublicDTO coursePublicDTO = DTOBuilder.buildDtoForEntity(course, CoursePublicDTO.class, selfLink);
+        return new ResponseEntity<>(coursePublicDTO, HttpStatus.OK);
     }
 
     @Auditable(action = AuditingAction.DELETE_COURSE)
-    @DeleteMapping(value = "/api/user/global/courses/{course_id}")
-    public void deleteGlobalCourse(@PathVariable Long course_id) throws NotAuthorisedUserException {
-        courseService.deleteGlobalCourse(course_id);
+    @DeleteMapping(value = "/api/cabinet/global/courses/{courseId}")
+    public ResponseEntity deleteGlobalCourse(@PathVariable Long courseId) throws NotAuthorisedUserException {
+        courseService.deleteGlobalCourse(courseId);
+        return new ResponseEntity(HttpStatus.OK);
     }
 
-    @DeleteMapping(value = "/api/user/local/courses/{course_id}")
-    public void deleteLocalCourse(@PathVariable Long course_id) throws NotAuthorisedUserException {
-        courseService.deleteLocalCourse(course_id);
+    @DeleteMapping(value = "/api/cabinet/local/courses/{courseId}")
+    public ResponseEntity deleteLocalCourse(@PathVariable Long courseId) throws NotAuthorisedUserException {
+        courseService.deleteLocalCourse(courseId);
+        return new ResponseEntity(HttpStatus.OK);
     }
 
     @Auditable(action = AuditingAction.ADD_COURSE)
-    @PutMapping("/api/user/courses/{course_id}")
-    public ResponseEntity addCourse(@PathVariable Long course_id) throws NotAuthorisedUserException {
-        Course course = courseService.updateListOfCoursesOfTheAuthorizedUser(course_id);
-        return new ResponseEntity(HttpStatus.OK);
+    @PostMapping("/api/cabinet/courses/{courseId}")
+    public ResponseEntity<CoursePublicDTO> addCourse(@PathVariable Long courseId) throws NotAuthorisedUserException {
+        Course course = courseService.updateListOfCoursesOfTheAuthorizedUser(courseId);
+        Link selfLink = linkTo(methodOn(CourseController.class).getCourseById(course.getId())).withSelfRel();
+        CoursePublicDTO coursePublicDTO = DTOBuilder.buildDtoForEntity(course, CoursePublicDTO.class, selfLink);
+        return new ResponseEntity<>(coursePublicDTO, HttpStatus.OK);
     }
 
     @Auditable(action = AuditingAction.ADD_COURSE)
-    @PutMapping("/api/user/courses/{course_id}/update/access")
-    public ResponseEntity updateCourseAccess(@PathVariable Long course_id,
+    @PutMapping("/api/cabinet/{courseId}/update/access")
```

Comment:
We can add course to cabinet by url `/api/cabinet/courses/{courseId}` (addCourse method).
When we want to update course access we use next url `/api/cabinet/{courseId}/udpate/access`.
Maybe better to use link `/api/cabinet/courses/{courseId}` to update course access, because, access is a property of course.